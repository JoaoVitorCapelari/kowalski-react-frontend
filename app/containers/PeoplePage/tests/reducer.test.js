import peoplePageReducer from '../reducer';

describe('peoplePageReducer', () => {
  it('returns the initial state', () => {
    expect(peoplePageReducer(undefined, {})).toMatchSnapshot();
  });
});
