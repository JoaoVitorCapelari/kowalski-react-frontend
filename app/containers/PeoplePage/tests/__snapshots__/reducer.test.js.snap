// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`peoplePageReducer returns the initial state 1`] = `
Immutable.Map {
  people: Immutable.List [
  ],
  personFormOpen: false,
  isLoadingPeople: false,
  loadingPeopleErrorMsg: "",
  isFetchingNewPersonFormOptions: false,
  roles: Immutable.List [
  ],
}
`;
